static/src/js/components/PortalBrowserModal/__tests__/PortalList.test.js

```diff
@@ -59,44 +68,85 @@ describe('PortalList component', () => {
     const { history, props } = setup()
 
     const portalTitle = screen.queryByTestId('portal-list-item-standardproducts')
+
     await user.click(portalTitle)
 
-    expect(history.location).toEqual(expect.objectContaining({
-      pathname: '/search',
-      search: '?portal=standardproducts'
-    }))
+    await waitFor(() => {
+      expect(history.location).toEqual(expect.objectContaining({
+        pathname: '/search',
+        search: '?portal=standardproducts'
+      }))
+    })
 
     expect(props.onModalClose).toHaveBeenCalledTimes(1)
   })
 
   test('clicking on the `More Info` does not open the portal', async () => {
-    const user = userEvent.setup()
-
+    const user = await userEvent.setup()
     const { history, props } = setup()
 
     const moreInfoLink = screen.getByTestId('portal-link-above').querySelector('a')
+
     await user.click(moreInfoLink)
 
-    expect(history.location).toEqual(expect.objectContaining({
-      pathname: '/search'
-    }))
+    await waitFor(async () => {
+      expect(history.location).toEqual(expect.objectContaining({
+        pathname: '/search'
+      }))
 
-    expect(props.onModalClose).toHaveBeenCalledTimes(0)
+      expect(props.onModalClose).toHaveBeenCalledTimes(0)
+    })
   })
 
   test('displays a title attribute for the `More Info` link', async () => {
     setup()
 
     const moreInfoLink = screen.getByTestId('portal-link-above').querySelector('a')
 
-    expect(moreInfoLink).toHaveAttribute('title', 'Find more information about ABoVE (Arctic-Boreal Vulnerability Experiment)')
+    await waitFor(() => {
+      expect(moreInfoLink).toHaveAttribute('title', 'Find more information about ABoVE (Arctic-Boreal Vulnerability Experiment)')
+    })
   })
 
-  test('Clicking the `More Info` link opens a new windoe', async () => {
+  test('Clicking the `More Info` link opens a new window', async () => {
     setup()
 
     const moreInfoLink = screen.getByTestId('portal-link-above').querySelector('a')
 
-    expect(moreInfoLink).toHaveAttribute('target', '_blank')
+    await waitFor(() => {
+      expect(moreInfoLink).toHaveAttribute('target', '_blank')
+    })
+  })
+
+  describe('When loading a portal without the portalBrowser flag', () => {
```

Comment:
portalBrowser is defaulting to false correct so that it has to be explicitly added into the config for it to show up in the modal?